runner/runners/README.md

```diff
@@ -0,0 +1,29 @@
+# Runner Impl Design #
+This package contains implementations of Runner.
+
+There are three main components of our Runner Impl:
+* Statuses
+* Controller
+* Invoker
+
+## Statuses (statuses.go) ##
+Statuses is a database of ProcessStatus. It has two facets:
+* Read Statuses (by implementing runner.Statuser). Allows reading the current state and waiting for new results.
+* Write Statuses. What writes Statuses?
```

Comment:
Ugh, I was trying to lead into the next paragraph but it was unclear. Rephrased. How's that?